src/proxy/processors/push-action/scanDiff.js

```diff
@@ -0,0 +1,99 @@
+const Step = require('../../actions').Step;
+const config = require('../../../config');
+
+const commitConfig = config.getCommitConfig();
+const privateOrganizations = config.getPrivateOrganizations();
```

Comment:
These are new config sections. Can we add some example inputs or defaults into the [proxy.config.json](https://github.com/finos/git-proxy/blob/b326e26adf03475d8611c1b6d8f2a3c04d56ad51/proxy.config.json) at the repo root? Below patch is what I applied while testing.

```diff
diff --git a/proxy.config.json b/proxy.config.json
index 683fdf7..df8d6df 100644
--- a/proxy.config.json
+++ b/proxy.config.json
@@ -6,18 +6,36 @@
   },
   "authorisedList": [
     {
-      "project": "finos",
-      "name": "git-proxy",
-      "url": "https://github.com/finos/git-proxy.git"
+      "project": "coopernetes",
+      "name": "backstage",
+      "url": "https://github.com/coopernetes/backstage.git"
     }
   ],
+  "commitConfig": {
+    "message": {
+      "block": {
+        "literals": ["foo"],
+        "patterns": ["bar"]
+      }
+    },
+    "author": {
+      "email": {
+        "domain": {
+          "allow": ".*"
+        },
+        "local": {
+          "block": "foo"
+        }
+      }
+    }
+  },
   "sink": [
     {
       "type": "fs",
       "params": {
         "filepath": "./."
       },
-      "enabled": true
+      "enabled": false
     },
     {
       "type": "mongo",
@@ -25,7 +43,7 @@
       "options": {        
         "useUnifiedTopology": true
       },      
-      "enabled": false
+      "enabled": true
     }
   ],
   "authentication": [
diff --git a/src/config/index.js b/src/config/index.js
index e94f38c..c2b2db1 100644
--- a/src/config/index.js
+++ b/src/config/index.js
@@ -71,7 +71,37 @@ const logConfiguration = () => {
 };
 
 // logConfiguration();
-
+module.exports = {
+  getCommitConfig : () => {
+    return defaultSettings.commitConfig;
+  },
+  getPrivateOrganizations : () => {
+    return defaultSettings.privateOrganizations;
+  },
+  getAuthorisedList,
+  getDatabase,
tom@duncan:~/repos/git-proxy$ git diff src/config
diff --git a/src/config/index.js b/src/config/index.js
index e94f38c..c2b2db1 100644
--- a/src/config/index.js
+++ b/src/config/index.js
@@ -71,7 +71,37 @@ const logConfiguration = () => {
 };
 
 // logConfiguration();
-
+module.exports = {
+  getCommitConfig : () => {
+    return defaultSettings.commitConfig;
+  },
+  getPrivateOrganizations : () => {
+    return defaultSettings.privateOrganizations;
+  },
+  getAuthorisedList,
+  getDatabase,
+  logConfiguration,
+  getAuthentication,
+  getTempPasswordConfig,
+  getAllowSelfSignedCert : () => {
+    return true;
+  },
+  getProxyUrl : () => {
+    if (defaultSettings.proxyUrl) {
+      return defaultSettings.proxyUrl;
+    }
+    return 'https://github.com'
+  },
+  getCookieSecret : () => {
+    if (defaultSettings.cookieSecret) {
+      return defaultSettings.cookieSecret;
+    }
+    return 'secret';
+  },
+  getSessionMaxAgeHours : () => {
+    return 24;
+  },
+}
 exports.getAuthorisedList = getAuthorisedList;
 exports.getDatabase = getDatabase;
 exports.logConfiguration = logConfiguration;
```

Reply:
@coopernetes - resolved 👍